chef/cookbooks/neutron/libraries/default.rb

```diff
@@ -0,0 +1,19 @@
+def is_crowbar?()
+  return defined?(Chef::Recipe::Barclamp) != nil
+end
+
+def get_network_nodes()
+  if is_crowbar?
```

Comment:
Do we really need this? I know we use this in cookbooks which are usable without Crowbar, but the neutron cookbook currently is not one of them. I am not strictly opposed to adding this here, but it seem weird to me and it certainly doesn't make the cookbook crowbar independent. :)